src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -136,11 +136,11 @@ def save_pretrained_wrapper(
             # copy python files from cache dir to save_path if any
             copy_python_files_from_model_cache(model, save_directory)
 
-        save_pretrained_wrapper._overriden = True
+        save_pretrained_wrapper._overridden = True
         return save_pretrained_wrapper
 
     # wrap save_pretrained if not already
-    if not getattr(model.save_pretrained, "_overriden", False):
+    if not getattr(model.save_pretrained, "_overridden", False):
```

Comment:
Don't we still need to check for `_overriden` for models that were saved previous to this change?